.github/workflows/cmake.yml

```diff
@@ -744,6 +744,7 @@ jobs:
     name: Upload Coverage Reports
     runs-on: ubuntu-latest
     needs: cmake
+    if: ${{ always() }}
```

Comment:
In our work repos we have recently changed it to:

cancelled() == false

This is essentially same as always() except it doesn't do it if a workflow has been manually cancelled.

== false is necessary because linters complain about ! at the start of a value.

Reply:
Sounds like a good idea, lets test it 👍